asv_bench/benchmarks/pandas/indexing.py

```diff
@@ -0,0 +1,296 @@
+"""
+These benchmarks are for Series and DataFrame indexing methods.  For the
+lower-level methods directly on Index and subclasses, see index_object.py,
+indexing_engine.py, and index_cached.py
+"""
+import warnings
+
+import numpy as np
+
+from modin.pandas import (
+    DataFrame,
+    Float64Index,
+    Int64Index,
+    MultiIndex,
+    Series,
+    UInt64Index,
+    concat,
+    date_range,
+    period_range,
+)
+
+import pandas._testing as tm
+
+
+class NumericSeriesIndexing:
```

Comment:
As far as I understand we have some of these already, let's leave only really valuable ones

Reply:
Please be more specific.

Reply:
My concern is only to avoid benchmarking duplication

Reply:
I let @YarShev and @anmyachev decide whether we can/should skip some cases

Reply:
I would suggest to integrate new tests directly into our asv benchmarking system so we can see the difference and new added cases.